### PR TITLE
Task Page

### PR DESCRIPTION
**Pull request by SukanyaSen530**
Page -
- Task Page
- Layout Page

Components - 
- TaskForm Modal
- Task Card
- InputField - support for textarea

Task context, actions, reducers

Functionalities -
- View all tasks
- Create a task
- Delete a task
- Update a task

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
looks good to be merged

File: src/pages/Layout/Layout.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import "./layout.scss";
+
+const Layout = ({ renderHeader = false, header, children }) => {
+  return (
+    <section className="layout flex flex-col">
+      {renderHeader ? (
+        <div className="layout__header b-margin-md"> {header} </div>
+      ) : null}
+      <div className="layout__body">{children}</div>
+      {/* <button className="btn">Logout</button> */}
```
Comment (prathmeshjagtap) [inline src/pages/Layout/Layout.jsx:10]: you can remove commented code if not required

**Review by VedanthB — APPROVED** [review]
Good work 

things i liked
- the code is clean and structured 
- use of sass

you can merge the PR

**Review by prathmeshjagtap — APPROVED** [review]
Code is Well structured
I liked you have divided code into small files 
it becomes easy to understand

File: src/pages/Layout/Layout.jsx (hunk 1 above)
Comment (SukanyaSen530) [reply src/pages/Layout/Layout.jsx:10]: I will use it in next PR :)

**Pull request merged**